modules/pubsub/src/main/resources/reference.conf

```diff
@@ -2,10 +2,17 @@ snowplow.defaults: {
   sources: {
     pubsub: {
       parallelPullFactor: 0.5
-      durationPerAckExtension: "60 seconds"
+      durationPerAckExtension: "15 seconds"
```

Comment:
I decreased this because I saw in the latency metrics some rare spikes to 60 seconds, presumably caused by a failed ack (??).  Reducing this value will reduce the size of the latency spike if that ever happens again.  It doesn't need to be as high as 60 seconds for our streaming applications.